lib/colorls/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module ColorLS
-  VERSION = '1.5.0'
+  VERSION = '1.5.1'
```

Comment:
We didn't release version 1.5.0 yet, so no need to bump the version.

Reply:
Fixed

Reply:
@avdv Do you need help in fixing the release script? By not releasing these latest changes, we are robbing people of all the new features being created

Reply:
Thank you, but I already created a PR which fixes the problem. But I cannot merge it without an approval from another maintainer.